Makefile

```diff
@@ -227,3 +227,36 @@ endif
 
 	docker run $(MODE) -p $(HOST_PORT):8545 -p $(HOST_METRICS_PORT):8080 $(MOUNT) "$(CONTAINER_REGISTRY)/flow-evm-gateway:$(IMAGE_TAG)" $(CMD_ARGS)
 
+.PHONY: start-testnet
```

Comment:
Can you add a comment that the key is from the emulator account (so it doesn't freak people out 😄)

Reply:
Removed them altogether in https://github.com/onflow/flow-evm-gateway/pull/728/commits/91b1ec08937541249f31e7ddbf330e67afdaf8b0 . They are not really relevant to the PR.